### PR TITLE
OKTA-461493: add email templates APIs

### PR DESCRIPTION
**Pull request by bryanapellanes-okta**
- OKTA-461493: regenerate from spec to add email template related endpoints
- OKTA-461493: add email template unit and integration tests

### CODE REVIEW

File: src/Okta.Sdk.IntegrationTests/BrandScenarios.cs [hunk 1]
```diff
@@ -150,5 +147,254 @@ public async Task UpdateBrandTheme()
...
+
+            var emailTemplateCustomization = await client.Brands.CreateEmailTemplateCustomizationAsync(emailTemplateCustomizationRequest, brand.Id, emailTemplate.Name);
+            emailTemplateCustomization.Should().NotBeNull();
+            emailTemplateCustomization.Body.Should().Be(testBody);
+            emailTemplateCustomization.Subject.Should().Be(testSubject);
+            emailTemplateCustomization.Language.Should().Be(testLanguage);
+
+            if ((bool)!emailTemplateCustomization.IsDefault)
+            {
+                await client.Brands.DeleteEmailTemplateCustomizationAsync(brand.Id, emailTemplate.Name, emailTemplateCustomization.Id);
```
Comment (laura-rodriguez) [inline src/Okta.Sdk.IntegrationTests/BrandScenarios.cs:214]: Can we put the delete method inside a finally block?

File: src/Okta.Sdk.IntegrationTests/BrandScenarios.cs [hunk 2]
```diff
@@ -150,5 +147,254 @@ public async Task UpdateBrandTheme()
...
+            var retrievedEmailTemplateCustomization = await client.Brands.GetEmailTemplateCustomizationAsync(brand.Id, emailTemplate.Name, emailTemplateCustomization.Id);
+
+            retrievedEmailTemplateCustomization.Should().NotBeNull();
+            retrievedEmailTemplateCustomization.Body.Should().Be(testBody);
+            retrievedEmailTemplateCustomization.Subject.Should().Be(testSubject);
+            retrievedEmailTemplateCustomization.Language.Should().Be(testLanguage);
+
+            if ((bool)!retrievedEmailTemplateCustomization.IsDefault)
+            {
+                await client.Brands.DeleteEmailTemplateCustomizationAsync(brand.Id, emailTemplate.Name, retrievedEmailTemplateCustomization.Id);
```
Comment (laura-rodriguez) [inline src/Okta.Sdk.IntegrationTests/BrandScenarios.cs:289]: Same here, let's put this line inside a finally block

File: src/Okta.Sdk.UnitTests/Okta.Sdk.UnitTests.csproj [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>    
-    <TargetFrameworks>netcoreapp2.0;netcoreapp3.1;net5.0</TargetFrameworks>
```
Comment (laura-rodriguez) [inline src/Okta.Sdk.UnitTests/Okta.Sdk.UnitTests.csproj:3]: Why are we removing this?

Comment (bryanapellanes-okta) [reply src/Okta.Sdk.UnitTests/Okta.Sdk.UnitTests.csproj:3]: removed netcoreapp2.0 because it wasn't in the travis build environment.

**Review by laura-rodriguez — APPROVED** [review]

**Pull request merged**